cmd/init.go

```diff
@@ -986,6 +986,7 @@ var initCmd = &cobra.Command{
 			fmt.Printf("  Username: %s\n", curConfig.Capella.Username)
 			fmt.Printf("  Password: %s\n", strings.Repeat("*", len(curConfig.Capella.Password)))
 			fmt.Printf("  Organization ID: %s\n", curConfig.Capella.OrganizationID)
+			fmt.Printf("  Override Token: %s\n", curConfig.Capella.OverrideToken)
```

Comment:
Should this be protected like the password above?

Reply:
okay